tester/consumer.go

```diff
@@ -107,7 +107,16 @@ type partConsumerMock struct {
 func (pcm *partConsumerMock) catchup() int {
 	var numCatchup int
 	for _, msg := range pcm.queue.messagesFromOffset(pcm.hwm) {
+		headers := make([]*sarama.RecordHeader, len(msg.headers))
```

Comment:
Hi :)

To clarify, where is the `headers` list been used? I just see you are assigning `RecordHeader`'s to it but I don't see what's using `headers`. 

Also, this is similar to what you have in the `SaramaHeaders()` function from tester/queue.go (line 16).

Thanks!

Reply:
Removed from new commit